teradatalabs/mapr52-base/README.md

```diff
@@ -0,0 +1,9 @@
+# mapr52-base
```

Comment:
add the badges, as it is done for any other image. Just copy + paste then search & replace the image name